src/ipc/ipc-common/IPCProxyModelProperty.h

```diff
@@ -168,7 +167,10 @@ class IPCProxyModelProperty : public facelift::ModelProperty<ModelDataType>
             requestItemsAsync(requestMemberID, previousIndex);
         }
 
-        return retval;
+        if (m_cache.exists(row)) {
```

Comment:
Get rid of multiple return paths with :
return (m_cache.exists(row) ? m_cache.get(row) : ModelDataType {});

Reply:
pretty much no difference, IMO both versions look good

Reply:
ok, agree.
done